### PR TITLE
Add implementation of pushSame for array

### PR DESCRIPTION
**Pull request by belochub**
I've decided to create this function after benchmarking it against the alternatives:
https://jsbench.me/pzjsyqrz9h/1

### CODE REVIEW

File: lib/array.js [hunk 1]
```diff
@@ -64,10 +64,26 @@ const sequence = (seq, max) => {
 // Returns: <any>, element
 const last = arr => arr[arr.length - 1];
 
+// Push single value multiple times
+//   arr - <Array>
+//   n - <number>
+//   value - <any>
+// Returns: <number>, new value of arr.length
+const pushSame = (arr, n, value) => {
```
Comment (nechaido) [inline lib/array.js]: `pushMultiple`?

Comment (belochub) [reply lib/array.js]: @nechaido the function pushes the same value 0 or more times, thus I think naming it `pushMultiple` would be ambiguous.

Comment (nechaido) [reply lib/array.js]: @belochub I do not like `pushSame`, but I do not have any better option, so you can proceed and land it.

**Comment by belochub** [comment]
Landed in https://github.com/metarhia/common/commit/54731888dfcef45946cba5a671d18ea9a4998974.

**Pull request closed without merging**